### DISCUSSION TITLE
Support for later versions of auth?

### DISCUSSION THREAD

**Discussion by dpchappell** [prowide/prowide-iso20022#108] [Q&A] [answered]
Hi team

Is there any chance you can upgrade auth to include subsequent versions from the version that is packaged? In particular I am interested in auth.030.001.03 and auth.108.001.01

Thanks so much for any help you can provide.

Dave

**Comment by ptorres-prowide** [comment, marked as answer]
Hello @dpchappell,

We support the following:

https://github.com/prowide/prowide-iso20022/blob/main/model-auth-mx/src/generated/java/com/prowidesoftware/swift/model/mx/MxAuth03000103.java

We support from MxAuth030001 to MxAuth030004.

https://github.com/prowide/prowide-iso20022/blob/main/model-auth-mx/src/generated/java/com/prowidesoftware/swift/model/mx/MxAuth10800101.java

We support from MxAuth108001and MxAuth108002.

Does it help?
